src/main/java/com/github/kokorin/jaffree/JaffreeRuntimeException.java

```diff
@@ -0,0 +1,18 @@
+package com.github.kokorin.jaffree;
+
+public class JaffreeRuntimeException extends RuntimeException {
```

Comment:
Probably we need better name for the exception. May be just `JaffreeException` or `FFException` ?

Reply:
It would probably be better to call it JaffreeException, it's more unique compare to FFException that _might_ crash with what someone else is calling it.